Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -866,7 +866,8 @@ namespace O3DE::ProjectManager
             try
             {
                 projectInfo.m_projectName = Py_To_String(projectData["project_name"]);
-                projectInfo.m_displayName = Py_To_String_Optional(projectData, "display_name", projectInfo.m_projectName);
+                // set the project display name to a length of 22 characters if the name is longer than that
```

Comment:
[recommendation] I'd just say "Set the project display set will clamp the name to the appropriate length", that way if in the future the number total of characters changes the comment won't get out of date.